src/gc/windows/gcenv.windows.cpp

```diff
@@ -781,6 +798,23 @@ bool GCToOSInterface::VirtualCommit(void* address, size_t size, uint16_t node)
     }
 }
 
+// Reserve virtual memory range.
+// Parameters:
+//  size      - size of the virtual memory range
+//  alignment - requested memory alignment
+//  flags     - allocation flags
+//  node      - the NUMA node to reserve memory on
+// Return:
+//  Starting virtual address of the reserved range
+void* GCToOSInterface::VirtualReserveNuma(size_t size, size_t alignment, uint32_t flags, uint16_t node)
```

Comment:
Should we rather add the `node` argument to VirtualReserve instead of adding a new API here? 

It would make this consistent with `GCToOSInterface::VirtualCommit` - that has `node` argument as well.

Reply:
yes